packages/components/src/components/carousel/carousel.jsx

```diff
@@ -112,7 +112,7 @@ const Carousel = ({
 
                     {show_nav && nav_position === 'middle' && sliced_list_length > 1 && (
                         <span className='dc-carousel__icon' onClick={handleNextClick}>
-                            <Icon icon='IcChevronRight' size='24' />
+                            <Icon icon='IcChevronRightBoldMt5' size='28' />
```

Comment:
Changing these icons inside `Carousel` affects every use-case of `Carousel` in the whole project. are you sure we want this? if we Only want to change these icons in one place we have to add a prop and use that prop in that location only